subprojects/docs/src/docs/userguide/dep-man/06-publishing/publishing_ivy.adoc

```diff
@@ -182,22 +182,3 @@ The result is that the following artifacts will be published for each project:
 * The Ivy module metadata file: `ivy-1.0.xml`.
 * The primary JAR artifact for the Java component: `project1-1.0.jar`.
 * The Javadoc and sources JAR artifacts of the Java component (because we configured `withJavadocJar()` and `withSourcesJar()`): `project1-1.0-javadoc.jar`, `project1-1.0-source.jar`.
-
-[[sec:pub_ivy:legacy_migration]]
-== Migrating from legacy Ivy publication
```

Comment:
Why remove this now?
Devs might still need to be aware of these differences when they attempt to migrate.

Reply:
Because the documentation page it references uses the old publishing mechanism in samples, which is not available. The migration path is explained in 6.x and in the upgrade guide (users had the 6.x line to do the transition).

Reply:
Ok with the removal.

However the disappearance of default upload archives should be referenced in `upgrading_version_6.adoc`